### ISSUE TITLE
Any news about supporting Rails 6?

### ISSUE THREAD

**Issue by SerhiiVynohradov** [bborn/communityengine#334]
I tried it and got this:
````bash
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 6.1.3.2)

  In Gemfile:
    rails (~> 6.1.3, >= 6.1.3.2)

    community_engine was resolved to 2.0.0.beta2, which depends on
      rails (= 3.1.2)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

Bundler could not find compatible versions for gem "sass-rails":
  In snapshot (Gemfile.lock):
    sass-rails (= 6.0.0)

  In Gemfile:
    sass-rails (>= 6)

    community_engine was resolved to 2.3.2, which depends on
      sass-rails (~> 3.2.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
````

Tl dr: it needs to at least update sass-rails before it's usable.

**Comment by bborn** [comment]
Sorry - I'm not updating this gem anymore. Feel free to fork and update.
